### DISCUSSION TITLE
Url with github pages contains github.io/github.io/

### DISCUSSION THREAD

**Discussion by matthewjohnsondurham** [community/community#23408] [Other Feature Feedback, Questions, & Ideas]
<p>I am trying to publish with github pages.  I created the repository github dot io (forum rules limit how many links my post can include) and added a test page index dot html</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://github.com/matthewjohnsondurham/github.io">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">

      <a href="https://github.com/matthewjohnsondurham/github.io" target="_blank" rel="noopener nofollow ugc">GitHub</a>
  </header>

  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/345;"><img src="https://user-images.githubusercontent.com/43553515/181076781-18d56cc9-0a3e-4b73-a611-737078f6fdfb.png" class="thumbnail" width="690" height="345" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/4/4/448bd3e78c813615fd01a06940313c369b52d6c6_2_690x345.png, https://aws1.discourse-cdn.com/github/optimized/3X/4/4/448bd3e78c813615fd01a06940313c369b52d6c6_2_1035x517.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/4/4/448bd3e78c813615fd01a06940313c369b52d6c6.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/4/4/448bd3e78c813615fd01a06940313c369b52d6c6_2_10x10.png"></div>

<h3><a href="https://github.com/matthewjohnsondurham/github.io" target="_blank" rel="noopener nofollow ugc">matthewjohnsondurham/github.io</a></h3>

  <p>Contribute to matthewjohnsondurham/github.io development by creating an account on GitHub.</p>

  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p>and then published via Settings-&gt;Pages but this creates the url</p>
<p><a href="https://matthewjohnsondurham.github.io/github.io/" class="onebox" target="_blank" rel="noopener nofollow ugc">https://matthewjohnsondurham.github.io/github.io/</a></p>
<p>The extra github dot io is unexpected.</p>
<p>What I have done wrong, and how can I create the desired url?</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>To create a user page (without the repository name in the path) your repository needs to be named with the pattern <code>&lt;username&gt;.github.io</code>, so in your case <code>matthewjohnsondurham.github.io</code>, not just <code>github.io</code>.</p>
<p>See: <a href="https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages#types-of-github-pages-sites" class="inline-onebox">About GitHub Pages - GitHub Docs</a></p>

**Comment by matthewjohnsondurham** [comment]
<p>Thanks a lot – wasn’t reading the docs carefully enough</p>
